worker/src/main/java/org/apache/celeborn/service/deploy/worker/memory/MemoryManager.java

```diff
@@ -293,6 +297,18 @@ public boolean shouldEvict(boolean aggressiveMemoryFileEvictEnabled, double evic
 
   public ServingState currentServingState() {
     long memoryUsage = getMemoryUsage();
+    long allocatedMemory;
+    if (networkMemoryAllocatorPooled) {
+      allocatedMemory = getAllocatedMemory();
+    } else {
+      allocatedMemory = memoryUsage;
+    }
+    // trigger resume
+    // CELEBORN-1792: resume should use pinnedDirectMemory instead of usedDirectMemory
```

Comment:
Although we needn't  change to pause state, it would be better to call trim when netty direct memory used above pausePushDataThreshold/pauseReplicateThreshold, WDYT?